Sources/Security.php

```diff
@@ -1475,7 +1475,7 @@ function corsPolicyHeader($set_header = true)
 			if (preg_match('~' . $host_regex . '~u', $origin['host']))
 			{
 				$context['cors_domain'] = trim($_SERVER['HTTP_ORIGIN']);
-				$context['valid_cors_found'] = $origin['type'];
+				$context['valid_cors_found'] = !empty($allowed_origin['type']) ? $allowed_origin['type'] : true;
```

Comment:
Yes, this was supposed to be `$allowed_origin['type']` rather than `$origin['type']`.

However, I don't think the ternary is necessary. There's no way that `$allowed_origin['type']` can ever be empty here.

Reply:
Well was just making sure it would always validate.  Because valid_cors_found was null, the headers where not getting set.